vault/dendron.alternatives.md

```diff
@@ -1,18 +1,24 @@
 ---
 id: b33d1803-6e4e-418f-a269-c4d4bde967cf
 title: Dendron Alternatives
-desc: ''
-updated: 1616000570400
+desc: ""
+updated: 1622574631804
 created: 1615999699967
 nav_order: 5
 published: false
 ---
 
 The main difference between Dendron and almost every other note taking tool can be summed up below. This page is created to dive into additional details of specific tools and for SEO purposes 😅
 
-- NOTE: if you are one of the tools listed or would like to be, feel free to submit pull requests by following the process [[here|dendron.contribute.documentation]]
+-   NOTE: if you are one of the tools listed or would like to be, feel free to submit pull requests by following the process [[here|dendron.contribute.documentation]]
 
 ![[dendron.faq#how-is-dendron-different-from-x-note-taking-tool,1:#*]]
 
+All the tools below are excellent for their own use-cases. This comparison rests purely where Dendron is different.
 
-![[dendron.alternatives.*]]
\ No newline at end of file
+| Software Name | Comparison w/ Dendron                        | Flexible Hierarchy | Open source | Local-first | Fast and performant |
+| ------------- | -------------------------------------------- | ------------------ | ----------- | ----------- | ------------------- |
+| Dendron       | How?                                         | ✅                 | ✅          | ✅          | ✅                  |
+| Notion        | [[dendron.alternatives.notion-vs-dendron]]   | ✅                 | ❌          | ❌          | ✅                  |
+| Obsidian      | [[dendron.alternatives.obsidian-vs-dendron]] | ❌                 | ❌          | ✅          | ✅                  |
+| Roam          | [[dendron.alternatives.roam-vs-dendron]]     | ❌                 | ❌          | ❌          | ❌                  |
```

Comment:
this is a bit unfair to roam. something I would add to feature list is perhaps daily journals and bidirectional links

Reply:
also zoomable outlining (not sure what the term is), which only roam supports from this list at this point

Reply:
Good call out - Notion can also be used for outlining/page within a page so I gave it a green checkmark as well.

Reply:
@kevinslin Unless I misunderstanding what you mean by zoomable outlining — I haven't used Roam — at this point Obsidian has support for it through the [Outliner](https://github.com/vslinko/obsidian-outliner) and [Zoom](https://github.com/vslinko/obsidian-zoom) plugins.